src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -13,6 +13,9 @@
     public static final String MESSAGE_INVALID_ACTIVITY_DISPLAYED_INDEX = "The activity index provided is invalid";
     public static final String MESSAGE_INVALID_FRIEND_DISPLAYED_INDEX = "The friend index provided is invalid";
     public static final String MESSAGE_INVALID_TRAVELPLAN_DISPLAYED_INDEX = "The travel plan index provided is invalid";
+    public static final String MESSAGE_INVALID_SHOW_AT_WISHLIST = "There is no accommodation or friends in wishlist!";
```

Comment:
Maybe can change to "The wishlist only stores activities, not accommodations or friends!". If not it sounds like the accommodations and friends of wishlist is empty.

Reply:
Resolved.